napari/_qt/qt_main_window.py

```diff
@@ -874,11 +1025,29 @@ def activate(self):
         self._qt_window.raise_()  # for macOS
         self._qt_window.activateWindow()  # for Windows
 
+    def _toggle_theme(self):
+        self.qt_viewer.viewer._toggle_theme()
+        SETTINGS.application.theme = self.qt_viewer.viewer.theme
+        self._update_theme()
+
     def _update_theme(self, event=None):
         """Update widget color theme."""
         theme_name = self.qt_viewer.viewer.theme
         self._qt_window.setStyleSheet(get_stylesheet(theme_name))
 
+        if event:
+            SETTINGS.application.theme = event.value
+
+        # set window styles which don't use the primary stylesheet
+        theme = get_theme(SETTINGS.application.theme)
+        self._status_bar.setStyleSheet(
```

Comment:
this was also removed in #2263.  git-merge error?

Reply:
Indeed. Fixing˜